Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -743,7 +744,23 @@
                         .withId("note")
                         .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
                         .withContentOverflow()
-                        .withSourceValueEchoRendering());
+                        .withSourceValueEchoRendering())
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("LifeCycleStatus")
+                        .withTitle("Status")
+                        .withId("LifeCycleStatus")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
+                        .withFixedValueRange(lifeCycleStatusOptions.options.map(value => {
+                            return {
+                                textValue: value.text,
+                                remoteValue: value.id
+                            }
+                        })
+                        , false)
+                        .withRendering(dataItem => lifeCycleStatusOptions.mapValueFromString(dataItem.LifeCycleStatus) )
+                        .withExcelOutput(dataItem => lifeCycleStatusOptions.mapValueFromString(dataItem.LifeCycleStatus))
```

Comment:
no need to duplicate the rendering for excel if you want the same rendering as in the view...